lnrpc/stateservice.ts

```diff
@@ -259,24 +249,20 @@ export const GetStateResponse = {
           break;
       }
     }
-
     return message;
   },
 
   fromJSON(object: any): GetStateResponse {
-    const message = createBaseGetStateResponse();
-    message.state =
-      object.state !== undefined && object.state !== null
-        ? walletStateFromJSON(object.state)
-        : 0;
-    return message;
+    return {
+      state: isSet(object.state) ? walletStateFromJSON(object.state) : 0,
+    };
   },
 
   toJSON(message: GetStateResponse): unknown {
-    const object: any = {};
+    const obj: any = {};
```

Comment:
And here

Reply:
This is autogenerated code, I don't want to change it.